projects/llm-semconv.md

```diff
@@ -78,4 +79,4 @@ Alternating weekly meetings to accommodate different time zones:
 
 ## Project Board
 
-To be established post-approval.
\ No newline at end of file
+To be established post-approval.
```

Comment:
is this additional change on purpose?

Reply:
This should be the purpose, as if I edit using vim, this line diff will be generated **automatically**. I think the previous commit was using VSCode or some other IDE and it did not contain newline at end of file.

Reply:
gotcha, new line at the end of the file is fine.